modules/auth/auth.controller.js

```diff
@@ -0,0 +1,296 @@
+// Package imports
+const nodemailer = require('nodemailer');
+const jwt = require('jsonwebtoken');
+const env = require('dotenv').config();
+const {default: to} = require('await-to-js');
+
+// Constant Imports
+const authConstants = require('./auth.constants');
+const globalConstants = require('../../constants/constants.js');
+
+// Model imports
+const User = require('./auth.model');
+
+// Logger import
+const logger = require('../../config/logger/logger');
+
+/**
+ * Created login controller to send response to the user on login
+ * @param {Object} req
+ * @param {Object} res
+ * @return {Object}
+ */
+const loginCtrl = async (req, res) => {
```

Comment:
```suggestion
const login = async (req, res) => {
```
The name should be proper. No need to add ctrl in the name as it is already in the controller file.

Reply:
Same for all functions in the controller files.

Reply:
Done